README.adoc

```diff
@@ -83,7 +83,7 @@ This script can take 2 parameters referring to the repository and the branch to
 
 [source,shell script]
 ----
-./run_tests_with_s2i.sh "https://github.com/snowdrop/rest-http-example" branch-to-test
+    ./run_tests_with_s2i.sh "https://github.com/snowdrop/cache-example" branch-to-test
```

Comment:
There is some spaces at the beginning, did you add it on purpose?

Reply:
No, that was not supposed to go there. Fixed.